doc/classes/TextEdit.xml

```diff
@@ -543,6 +550,13 @@
 				[b]Note:[/b] The Y position corresponds to the bottom side of the line. Use [method get_rect_at_line_column] to get the top side position.
 			</description>
 		</method>
+		<method name="get_previous_composite_character_column" qualifiers="const">
+			<return type="int" />
+			<param index="0" name="caret_index" type="int" default="0" />
+			<description>
+				Returns the correct column at the start of a composite character like ❤️‍🩹 (mending heart; Unicode: U+2764 U+FE0F U+200D U+1FA79) which is comprised of more than one unicode, if the caret is at the end of the composite character.
```

Comment:
> if the caret is at the end of the composite character.

Haven't tested but doesn't it still work if the caret is in the middle of a composite character as well?

Might want to also mention that if it isn't a composite character, it still gets the next/prev column.